runtime/src/bank.rs

```diff
@@ -4537,10 +4537,7 @@ impl Bank {
     }
 
     pub fn get_snapshot_storages(&self) -> SnapshotStorages {
-        self.rc
-            .get_snapshot_storages(self.slot())
-            .into_iter()
-            .collect()
+        self.rc.get_snapshot_storages(self.slot())
```

Comment:
so this was unnecessary in the first place.. https://github.com/solana-labs/solana/pull/8339/files#diff-ed47b4a0198313377e091bb3957bbbc63d937805426d1b2b6de39d0a50d32a0cR1870